app/frontend/styles/_application-card.scss

```diff
@@ -0,0 +1,70 @@
+.app-application-card {
+  border-bottom: 1px solid govuk-colour('mid-grey');
+  margin-bottom: 15px;
+  padding-bottom: 15px;
```

Comment:
Could these be `govuk-spacing` instead? (+ the rest of the file)

Reply:
👍